src/pages/FormPage.tsx

```diff
@@ -132,6 +151,13 @@ const FormPage = () => {
                 no-min-width
                 placeholder="Majorvägen 32"
                 helper={addressValidation ? "" : "Address is mandatory field!"}
+                ref={(element) => {
+                  element?.addEventListener('tdsInput', (event) => {
+                    const customEvent = event as CustomEvent;
+                    const { value } = customEvent.detail.target;
+                    setAddressValue(value)
+                  })
```

Comment:
```suggestion
                  element?.addEventListener('tdsInput', (event as CustomEvent) => {
                    setAddressValue(event.detail.target)
                  })
```

Not sure if this works but would remove a couple of lines of code